Indexer packages are not built due to an error in `get-versions-wazuh-indexer-plugins`

## Description

While analyzing https://github.com/wazuh/wazuh-qa-automation/issues/4574, it has been detected that the indexer packages are not built due to an error in the `get-versions-wazuh-indexer-plugins` stage:

```
Run echo "version=$(bash ./scripts/product_version.sh)" >> "$GITHUB_OUTPUT"
bash: ./scripts/product_version.sh: No such file or directory
bash: scripts/opensearch_version.sh: No such file or directory
Error: Process completed with exit code 127.
```

## Evidence

- indexer-rpm-x86_64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/18958531973
- indexer-deb-arm64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/18958532021
- indexer-deb-amd64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/18958531958
- indexer-rpm-aarch64 -> https://github.com/wazuh/wazuh-indexer/actions/runs/18958532241

--- a/.github/workflows/5_builderpackage_indexer.yml
+++ b/.github/workflows/5_builderpackage_indexer.yml
@@ -158,10 +158,10 @@ jobs:
       - name: Read versions from wazuh-indexer-plugins
         id: plugins_version
         run: |
-          echo "version=$(bash ./scripts/product_version.sh)" >> "$GITHUB_OUTPUT"
+          echo "version=$(bash ./tools/product_version.sh)" >> "$GITHUB_OUTPUT"
 
           # Extracts the Opensearch version
-          opensearch_version=$(bash scripts/opensearch_version.sh "setup")
+          opensearch_version=$(bash ./tools/opensearch_version.sh "setup")
           echo "opensearch_version=$opensearch_version" >> "$GITHUB_OUTPUT"
         
   get-versions-wazuh-indexer-reporting:

